examples/transmitter/swagger_server/test/test_jwt_encode.py

```diff
@@ -7,16 +7,16 @@
 import uuid
 
 from jwcrypto.jwk import JWK, JWKSet
-
-from py._path.local import LocalPath
 import pytest
-from _pytest.monkeypatch import MonkeyPatch
 
-from swagger_server import jwt_encode
+from swagger_server.events import (
+    Events, SecurityEvent, VerificationEvent
+)
+import swagger_server.jwt_encode as jwt_encode
 
 
 @pytest.fixture(autouse=True)
-def clear_cache() -> None:
+def clear_cache():
```

Comment:
It wouldn't hurt to put all of this type annotation back in (for this function and the others that were updated)

Reply:
Yes, I missed this one.